common/storybook/.storybook/preview.tsx

```diff
@@ -18,7 +18,7 @@ const setupMsw = (): void => {
         quiet: true,
         onUnhandledRequest(request, print) {
             // MSW warns on all unhandled requests, but we don't necessarily care
-            const pathAllowList = ['/images/']
+            const pathAllowList = ['/images/', '/static/hedgehog/']
```

Comment:
this warns in the console multiple times but we don't care that it isn't mocked... so suppres it